src/components/auth/register/RegisterForm.tsx

```diff
@@ -36,15 +39,41 @@ const defaults: RegisterFormData = {
 export type RegisterFormProps = { initialValues?: RegisterFormData }
 
 export default function RegisterForm({ initialValues = defaults }: RegisterFormProps) {
-  const { keycloak } = useKeycloak<KeycloakInstance>()
+  const router = useRouter()
+  const { t } = useTranslation()
+  const [loading, setLoading] = useState(false)
+  const { mutateAsync: register } = useRegister()
 
   const onSubmit = async (values: RegisterFormData) => {
-    const result = await keycloak?.register({
-      loginHint: values.email, // Doesn't work with registration, see https://stackoverflow.com/q/63117669/668245
-      redirectUri: `${baseUrl}${routes.profile.index}`,
-    })
-    console.log(values)
-    console.log({ result })
+    AlertStore.show(t('auth:alerts.welcome'), 'success')
```

Comment:
This should probably not be here since down in the `resp?.ok` we have an alert showing welcome. I am not sure about that though